tests/e2e/test_diffusion.py

```diff
@@ -2611,7 +2611,7 @@ def test_ic_light(
         negative_text="lowres, bad anatomy, bad hands, cropped, worst quality",
     )
     ic_light_condition = sd.compute_gray_composite(image=kitchen_dog, mask=kitchen_dog_mask.convert("L"))
-    sd.set_ic_light_condition(ic_light_condition)
+    sd.set_ic_light_condition(ic_light_condition, use_rescaled_image=True)
```

Comment:
This flag (and corresponding doc) might even been removed: @Laurent2916 to confirm